packages/devextreme/js/viz/core/series_family.js

```diff
@@ -356,7 +356,7 @@ function updateRangeSeriesValues() {
         const minShownBusinessValue = minBarSize && valueAxisTranslator.getMinBarSize(minBarSize);
         if(minShownBusinessValue) {
             _each(singleSeries.getPoints(), function(_, point) {
-                if(!point.hasValue()) {
+                if(!point.hasValue() || that.rotated) {
```

Comment:
This looks strange to me, please clarify, what does the 'rotated' option have to do with this bug? and won't it affect scenarios when rotated is true, but we need to set minBarSize?

Reply:
I found out that the issue only happens when on rangeBar when rotated is true and minBarSize is set. Issue doesn't happen when rotated is set to false and minBarSize is set.

Reply:
i left a comment on teams channel